app/controllers/attachment_references_controller.rb

```diff
@@ -10,7 +10,7 @@ def create
   end
 
   def show
-    redirect_to @attachment_reference.url(filename: @attachment_reference.name)
+    render json: { redirectUrl: @attachment_reference.url(filename: @attachment_reference.name) }
```

Comment:
Dont forget to redirect in the frontend side

Reply:
There's actually nothing wrong about having `redirect_to` here. Attachments will ultimately go to `/uploads/*` anyway. It's the reverse proxy's job to proxy `/attachments/*` and `/uploads/*` to the server for downloads to work.